scripts/update-version.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/python3
+# This script will to help to manage rhpam components modules version, it will update all needed files
+# Example of usage:
+#   # move the current version to the next one or rcX
+#   python scripts/update-version.py -v 7.15.1
```

Comment:
change to python3 and maybe mention this is a dry-run and user would need to add `--confirm` to actually change

Reply:
sure thing, updated.